ci/aws-analytical-env/shared/meta.yml

```diff
@@ -214,3 +214,46 @@ meta:
       params:
         TF_CLI_ARGS_plan: -lock-timeout=300s
         TF_INPUT: "false"
+
+    vcs-to-s3:
+      task: vcs-to-s3
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: ((dataworks.docker_awscli_repository))
+            version: ((dataworks.docker_awscli_version))
+            tag: ((dataworks.docker_awscli_version))
+        params:
+          AWS_ROLE_ARN: arn:aws:iam::((aws_account.development)):role/ci
```

Comment:
you can remove this line, you are overriding it during the Spruce merge anyway

Reply:
removed it mirror-git-repo-s3.yml file